ckanext/geocat/harvester.py

```diff
@@ -422,3 +427,15 @@ def _get_object_extra(self, harvest_object, key):
 
 class GeocatConfigError(Exception):
     pass
+
+
+def _get_swisstopo_relation(pkg_id_swisstopo):
+    base_url = tk.config.get('ckanext.geocat.permalink_url', None)
+    bookmark = tk.config.get('ckanext.geocat.permalink_bookmark', None)
+    if base_url and bookmark:
+        bookmark_for_pkg = bookmark + pkg_id_swisstopo
+        permalink_title = tk.config.get(
+            'ckanext.geocat.permalink_title', base_url + bookmark_for_pkg)
```

Comment:
Can you explain why `ckanext.geocat.permalink_title` is preferred to `base_url + bookmark_for_pkg`?